src/watchtower.ts

```diff
@@ -0,0 +1,76 @@
+import { program } from "@commander-js/extra-typings";
+import { ReplayBlockOptions, ReplayTxOptions, RunOptions } from "./types";
+import { replayBlock, replayTx, run } from "./modes";
+
+async function main() {
+  program
+    .name("watchtower")
+    .description("Monitoring composable cows on the blockchain 🐮")
+    .version("0.2.0");
+
+  program
+    .command("run")
+    .description("Run the watchtower")
+    .requiredOption("--rpc <rpc...>", "Chain RPC endpoints to monitor")
+    .requiredOption(
+      "--contract <contractAddress>",
+      "ComposableCoW contract address"
+    )
+    .requiredOption(
+      "--deployment-block <deploymentBlock...>",
+      "Block number at which the contracts were deployed"
+    )
+    .requiredOption(
+      "--page-size <pageSize>",
+      "Number of events to fetch per page",
+      "5000"
```

Comment:
Given it has a default, does it need to be a required option?

Reply:
But is this the number of events or blocks?

Reply:
Made an `option`, and indeed it is related to blocks. Fixed.